tests/command/test_command_export.py

```diff
@@ -175,14 +178,22 @@ def test_export_groups(
 
 
 def test_export_includes_extras_by_flag(tester: CommandTester, do_lock: None) -> None:
-    tester.execute("--format requirements.txt --extras feature_bar")
+    tester.execute("--format requirements.txt --extras 'feature_bar feature_qux'")
```

Comment:
but now we lost the testcase for "exactly one extra at the command line" (and at the moment, that really is a separate code path)

Can you make this two testcases?  copy-paste is fine by me, `@pytest.mark.parametrize` (per a few other tests) if you feel like it and if it turns out nicely